tests/configuration/test_model.py

```diff
@@ -19,6 +20,106 @@ def test_disp_name_takes_precedence(self):
         assert dict_item.display_name == "a display"
 
 
+class TestEvent:
```

Comment:
Question: since you just added these tests, should there have been any tests deleted with the removal of DerivedFeature?

Reply:
Can you do a search for `DerivedFeature` and `_reduce_derived_features` to do some renames?

Reply:
Sadly(?) there weren't tests on this directly.  
I removed the docstring reference remaining on DerivedFeature-the class but left the static method as it's still useful to share the code in field_validators for cohorts and features